### PR TITLE
add real world fixtures

### PR DESCRIPTION
**Pull request by mapsam**
This lays the foundation for #49 by adding a `real-world/` directory. The extents and zooms (plus information about the area) are defined in `lib/real-world-extents.js` which is used to get tiles with a `process.env.MapboxAccessToken`. Also adds a REAL-WORLD.md file to get information about each extent set.

cc @joto @springmeyer @flippmoke @GretaCB

### CODE REVIEW

File: scripts/get-real-world.js [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+  console.log('bam.');
+});
+
+function getAndWriteTile(name, tileset, z, x, y, callback) {
+  let url = `https://api.mapbox.com/v4/${tileset}/${z}/${x}/${y}.vector.pbf?access_token=${process.env.MapboxAccessToken}`;
+  request.get(url, function(err, res, data) {
+    if (err) return callback(err);
+    if (res.statusCode !== 200) return callback(new Error(`status code error ${res.statusCode}`));
+    console.log(`writing ${name}/${z}-${x}-${y}.mvt`);
+    fs.writeFileSync(`/Users/mapsam/mapbox/mvt-fixtures/real-world/${name}/${z}-${x}-${y}.mvt`, data);
```
Comment (springmeyer) [inline scripts/get-real-world.js]: Should `fs.writeFileSync(`/Users/mapsam/mapbox/mvt-fixtures/real-world/${name}/${z}-${x}-${y}.mvt`, data);` be `fs.writeFileSync(__dirname + `./mvt-fixtures/real-world/${name}/${z}-${x}-${y}.mvt`, data);`?

File: scripts/get-real-world.js [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+const d3 = require('d3-queue');
+const fs = require('fs');
+const extents = require('../lib/real-world-extents.js');
+
+const sm = new SM();
+const q = d3.queue(5);
+
+for (let e in extents) {
+  let xyz = sm.xyz(extents[e].bbox, extents[e].zoom);
+  let count = 0;
```
Comment (springmeyer) [inline scripts/get-real-world.js]: Unused var?

File: scripts/get-real-world.js [hunk 3]
```diff
@@ -0,0 +1,40 @@
+'use strict';
+const request = require('request');
```
Comment (springmeyer) [inline scripts/get-real-world.js]: How about we move to https://github.com/sindresorhus/got, which is a smaller dep that request?

File: scripts/get-real-world.js (hunk 1 above)
Comment (mapsam) [reply scripts/get-real-world.js]: fixed!

File: scripts/get-real-world.js (hunk 2 above)
Comment (mapsam) [reply scripts/get-real-world.js]: removed

File: scripts/get-real-world.js (hunk 3 above)
Comment (mapsam) [reply scripts/get-real-world.js]: done

**Comment by mapsam** [comment]
@springmeyer I've incorporated your suggestions - a final 👁 would be great

**Comment by mapsam** [comment]
Per discussion with @springmeyer `got` doesn't seem to be working as we expect for this use-case. Moving back to `request` for now.

**Comment by mapsam** [comment]
Thank you @springmeyer. I think these will suffice for now - and we can add more in separate PRs.

**Pull request merged**